cfme-performance/utils/providers.py

```diff
@@ -407,6 +421,95 @@ def refresh_provider_vms_bulk(vm_ids):
         round(time.time() - starttime, 2)))
 
 
+def provision_vm(vm_name):
+    """Create a provision request for a VM"""
+    starttime = time.time()
+    data_dict = {
+        'action': 'create',
+        'resources': [{
+            'template_fields': {
+                'name': 'perf-provision-rhel7'
```

Comment:
I'm not quite sure what you mean by this. Do you mean that we will add templates to the providers within cfme_performance?